examples/node_labels.py

```diff
@@ -35,10 +35,16 @@ def main():
                 "baz": None}
         }
     }
-
-    api_response = api_instance.patch_node("minikube", body)
-
-    pprint(api_response)
+   
+    # Creating a list of cluster nodes
```

Comment:
this is just 'listing', not 'creating'

Reply:
Thank you @yliaog for pointing. I've improved the comment.